### DISCUSSION TITLE
Audit Log

### DISCUSSION THREAD

**Discussion by ryanalexander** [stoatchat/discussions#64] [Feature Suggestions]
Log all Administrative actions done in the server and display them in the audit log in the settings.

Should also have a rollback option for all settings for if someone tried to nuke the server

**Comment by Derpitron** [comment]
If possible, the user should be able to log different events/categories of events to different specialized channels, like a read-only text channel

**Reply by benjamin051000** [reply]
True. If bots are able to read server settings, they could just print an audit log into a read-only text channel. But it would probably be nicer if it was more integrated the way discord is.

**Comment by IAmTomahawkx** [comment]
Implemented and recording, pending frontend implementation.
